index.html

```diff
@@ -1,274 +1,379 @@
+		<link rel="stylesheet" href="./styles.css" />
+	</head>
+	<body>
+		<nav id="home">
+			<div class="mobile-menu-content">
+				<p class="logo"><a href="#home">My Logo</a></p>
+				<img class="hamburger" src="./assets/menu-icon.png" alt="menu" />
+			</div>
+			<div class="menu-bar">
+				<div class="menu-list">
+					<img
+						class="close"
+						src="./assets/icons8-close-64.png"
+						alt="close-icon"
+					/>
+					<ul class="nav-menu">
+						<li class="nav-list">
+							<a class="nav-link" href="#works">Portfolio</a>
+						</li>
+						<li class="nav-list">
+							<a class="nav-link" href="#about">About</a>
+						</li>
+						<li class="nav-list">
+							<a class="nav-link" href="#contact">Contact</a>
+						</li>
+					</ul>
+				</div>
+			</div>
+			<div class="desktop-menu-container">
+				<p class="logo"><a href="#home">My Logo</a></p>
+				<div class="desktop-link">
+					<ul>
+						<li><a class="nav-link" href="#works">Portfolio</a></li>
+						<li><a class="nav-link" href="#about">About</a></li>
+						<li><a class="nav-link" href="#contact">Contact</a></li>
+					</ul>
+				</div>
+			</div>
+		</nav>
+		<main>
+			<div class="header">
+				<h1>Hi, I'm David <br />Glad to see you!</h1>
+			</div>
+			<div class="text-body">
+				<p>
+					I’m a software developer! I can help you build a product, feature or
+					website. Look through some of my work and experience! If you like what
+					you see and have a project you need coded, don’t hestiate to contact
+					me.
+				</p>
+			</div>
+			<div class="lets-connect">
+				<h2>Let's Connect</h2>
+				<ul class="social-media-icons">
+					<li><img src="./assets/twitter.svg" alt="twitter-logo" /></li>
+					<li><img src="./assets/Linkedin icon.svg" alt="linkedin-logo" /></li>
+					<li><img src="./assets/medium.svg" alt="medium-logo" /></li>
+					<li><img src="./assets/github.svg" alt="github-logo" /></li>
+					<li><img src="./assets/peace-sign.svg" alt="angellist-logos" /></li>
+				</ul>
+			</div>
+		</main>
+		<section id="works">
+			<div class="display-card">
+				<div class="image-container">
+					<img src="./assets/Snapshoot Portfolio.png" alt="Snapshoot Portfolio" class="mobile" />
+					<img src="./assets/desktop/Snapshoot Portfolio(3).png" alt="Snapshoot Portfolio" class="desktop" />
+				</div>
+				<div class="text-container desktop">
+					<h1>Tonic</h1>
+					<div class="inline-text">
+						<h4>Canopy</h4>
+						<ul class="inline-list">
+							<li>Back End Dev</li>
+							<li>2015</li>
+						</ul>
+					</div>
+					<p class="body">
+						A daily selection of privately personalized reads; no accounts or
+						sign-ups required.
+					</p>
+					<ul class="stack">
+						<li class="border-background">html</li>
+						<li class="border-background">css</li>
+						<li class="border-background">javascript</li>
+					</ul>
+					<button class="button see-project" data-index="0">See Project</button>
+				</div>
```

Comment:
- [ ] As suggested by the previous reviewer that you should delete all the harcoded items from your html document and you should generate them from your javascript file 😊 Means you should have an array of objects containing information such as **Title, Description, Tech Stack, Project Image**, etc you can display that information on your information by injecting HTML code in your dom using javascript 👍

Hint: You can see this [pull request](https://github.com/HAMMAS-SALEEM/Portfolio/pull/16/files#diff-e07d531ac040ce3f40e0ce632ac2a059d7cd60f20e61f78268ac3be015b3b28f) to get an idea of what I am talking about 😊